src/components/ServiceMaturityReport.tsx

```diff
@@ -0,0 +1,238 @@
+import React from "react";
+import { Button, Grid } from "@material-ui/core";
+import { cloneDeep } from "lodash";
+import { useEntity } from "@backstage/plugin-catalog-react";
+import EntityOpsLevelMaturityProgress from "./EntityOpsLevelMaturityProgress";
+import CheckResultsByLevel from "./CheckResultsByLevel";
+import ServiceMaturitySidebar from "./ServiceMaturitySidebar";
+import {
+  LevelCategory,
+  LevelCheckResults,
+  LevelNode,
+  OpsLevelService,
+  OverallLevel,
+  ScorecardStats,
+} from "../types/OpsLevelData";
+
+type Props = {
+  exportEntity: (event: React.MouseEvent) => void;
+  exporting: boolean;
+  levels: LevelNode[];
+  service: OpsLevelService;
+  checkResultsByLevel: LevelCheckResults[] | undefined;
+  selectedCategories: string[];
+  scorecards: ScorecardStats[] | undefined;
+  overallLevel?: OverallLevel;
+  setSelectedCategories: (selectedCategories: string[]) => void;
```

Comment:
onCategorySelectionChange?

Reply:
Yeah, reasonable. I was trying to balance "change as little as possible" with "do it right", but happy to do it right!

Updating now.